docs/setup/independent/install-kubeadm.md

```diff
@@ -144,7 +144,7 @@ Docker. To ensure compatability you can either update Docker, like so:
 ```bash
 cat << EOF > /etc/docker/daemon.json
 {
-  "exec-opts": ["native.cgroupdriver=systemd"]
+  "exec-opt": ["native.cgroupdriver=systemd"]
```

Comment:
According to docker documentation here: https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file, the examples are showing that "exec-opts" is the correct key. Have you tried it yourself and verified if this change works?

Reply:
`exec-opts` is correct.

Reply:
In my test env with docker 1.12.6, exec-opt is correct. https://docs.docker.com/engine/reference/commandline/dockerd/ according to dockerd cmd option, it's also "exec-opt"

Reply:
unfortunately, there seem some inconsistency in docker documentation. when used in command line, the options is named `exec-opt`. however, when used in the json config file, the example shows `exec-opts`. it IS confusing.

Reply:
Here is json configuration, we should comply with json's format.

Reply:
But we con't start docker service with this json configuration, should we comply with a wrong configuration?

Reply:
@junxu Okay, if you have tested that `exec-opt` works and `exec-opts` doesn't. We can merge this.

Reply:
Sorry, exec-opts works well.